doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -39,6 +39,16 @@
 
 
 VENV_DIR_PREFIX = ".pyaedt_env"
+
+VENV_DIR = None  # "e:/pyaedt_env"
+if VENV_DIR:
+    pass
+elif is_windows:
```

Comment:
Just to avoid the first "if", I would do:

```suggestion
if not VENV_DIR and is_windows:
```

Reply:
> Just to avoid the first "if", I would do:

The first if is needed. Otherwise it will fail into Linux path.